tablet_integration_test.go

```diff
@@ -69,7 +69,9 @@ func TestTablets(t *testing.T) {
 			}
 		}
 
-		assertEqual(t, "queriedHosts", 1, queriedHosts)
+		if queriedHosts != 1 {
+			t.Errorf("trace should show only once host from the list %s, but it showed it %d, trace:\n%s", hostAddresses, queriedHosts, buf.String())
```

Comment:
you mean "one host" and "but it showed %d"? It is not about how many times it showed given host, but how many hosts it showed

Reply:
and I would consider using `Fatalf` to fail on first unsuccessful query

Reply:
I have changed language, take a lookl